app/carbonapi/app_test.go

```diff
@@ -64,7 +64,7 @@ func render(ctx context.Context, request types.RenderRequest) ([]types.Metric, e
 func renderErr(ctx context.Context, request types.RenderRequest) ([]types.Metric, error) {
 	return []types.Metric{
 		{},
-	}, errors.New("error during render")
+	}, fmt.Errorf("error during render")
```

Comment:
Again, nothing in terms of formatting and arguments. `errors.New` is the simpler alternative.

Reply:
Done!